File: workspaces/dotnet/dev-tools/src/DriverConfig.cs
using Newtonsoft.Json;

namespace OMP.LSWTSS;

public class DriverConfig
{
    [JsonProperty("engineAssemblyName")]
    public required string EngineAssemblyName { get; set; }

    [JsonProperty("engineAssemblyPath")]
    public required string EngineAssemblyPath { get; set; }

    [JsonProperty("engineAssemblyRuntimeConfigPath")]
    public required string EngineAssemblyRuntimeConfigPath { get; set; }

    [JsonProperty("engineClassName")]
    public required string EngineClassName { get; set; }
}

File: workspaces/rust/driver-library/src/driver_config.rs
#[derive(serde::Deserialize, serde::Serialize)]
#[serde(rename_all = "camelCase")]
pub struct DriverConfig
{
    pub engine_assembly_name: String,
    pub engine_assembly_path: String,
    pub engine_assembly_runtime_config_path: String,
    pub engine_class_name: String,
}
